runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -359,25 +359,20 @@ task validatesRunnerV2 {
     name: 'validatesRunnerV2Test',
     excludedCategories: [
       'org.apache.beam.sdk.testing.UsesRequiresTimeSortedInput',
```

Comment:
This should probably move to the `commonRunnerV2ExcludeCategories` as well since it applies to both streaming/batch.